### DISCUSSION TITLE
Need support from Handlebars to use in .Net framework 4.5 project

### DISCUSSION THREAD

**Discussion by sharadpise** [handlebars-lang/handlebars.js#1945] [General]
Hello Everyone,

I am trying to use handle bars in my project which target .net framework 4.5, But can't install it.. When I checked the supported target frameworks, last It is 4.5.1, I can not change my project target framework to 4.5.1.. Can any one help me make Handlebars support .Net framework 4.5..? TIA.

**Comment by jaylinski** [comment]
You can open an issue/discussion at https://github.com/Handlebars-Net/Handlebars.Net.

This repository is only for the JavaScript implementation of Handlebars.
